userguide/subscription/includes/overdue-overview.adoc

```diff
@@ -4,5 +4,7 @@ Kill Bill offers an `Overdue` feature, also referred to as Dunning, which is req
 * The `Overdue` module is in charge of altering the customer experience when payments are not met. This alteration may include some degradation or even cancelation of the service. The overdue state is defined at the account level, which means that if a user has multiple subscriptions, and an invoice associated with one of them has not been paid, the behavior of the system with regard to the overdue state will be global for all subscriptions.
 
 
-The Overdue module in Kill Bill relies on a configuration file which drives its behavior. The configuration can be specified at the tenant level, similarly to the catalog, or it can be specified globally for the system using the `org.killbill.overdue.uri` system property. The Overdue module listens to payment events, then changes the overdue state for an account to the appropriate state based on the configuration. Whether the payment was a failure or not, the Overdue module will recompute the state associated with the account and potentially make the transition to another state.
+The Overdue module in Kill Bill relies on a configuration file which drives its behavior. The configuration can be specified at the tenant level, similarly to the catalog, or it can be specified globally for the system using the `org.killbill.overdue.uri` system property. This property specifies an overdue configuration file `overdue.xml`. For details on this file see  https://docs.killbill.io/latest/overdue.html[Overdue System]. If no file is specified, default values will be used.
```

Comment:
@jdmooney That's good, but could we show an example of what the value is, e.g `org.killbill.overdue.uri=file:///<path>/overdue.xml` ?